openbb_platform/CONTRIBUTING.md

```diff
@@ -93,15 +124,15 @@ The standard models are defined under the `/OpenBBTerminal/openbb_platform/platf
 
 They define the [`QueryParams`](platform/provider/openbb_provider/abstract/query_params.py) and [`Data`](platform/provider/openbb_provider/abstract/data.py) models, which are used to query and output data. They are pydantic and you can leverage all the pydantic features such as validators.
 
-### Standardization Caveats
+##### Standardization Caveats
```

Comment:
4 (####) does not get included in the Table of Contents, at least the way we have things configured, it is limited to three.

Reply:
How did you test that?

![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/48914296/7da3c6b2-b91c-49a7-98bb-d2555842406e)